Library/UIBarButtonItem.swift

```diff
@@ -2,7 +2,7 @@ import class UIKit.UIBarButtonItem
 import struct ObjectiveC.Selector
 
 public extension UIBarButtonItem {
-  public static func back(_ target: AnyObject?, selector: Selector) -> UIBarButtonItem {
+  public static func back(_ target: AnyObject?, selector: Selector?) -> UIBarButtonItem {
```

Comment:
even tho we provide a `nil` selector when using this, it still operates as a back button to pop off the nav stack